aea/cli_gui/__init__.py

```diff
@@ -76,7 +67,7 @@ def read_description(dir_name, yaml_name):
             if "description" in yaml_data:
                 return yaml_data["description"]
         except yaml.YAMLError as exc:
-            print(exc)
+            logging.error(exc)
```

Comment:
add `logger = logging.gerLogger(__name__)` at the beginning of the module, then replace `logging.(\w)*` with `logger.$1` (regular expression notation)

Reply:
I don't understand the regular expression notation.

I saw a similar comment before I tried putting:
logger = logging.gerLogger(__name__)

At the top fo the file and then replacing the above line with:
logger.error(exc)

(or in other cases logger.info() )

And when I did this I no longer got login messages.